test_data/pil/sum_via_witness_query.pil

```diff
@@ -2,7 +2,8 @@ let N: int = 4;
 
 namespace std::prover(N);
     enum Query {
-        Input(int)
+        Input(int),
+        Unconstrained,
```

Comment:
Hmm, I think `Unconstrained` is misleading, it's not like otherwise any constraints are added! Maybe just `None`?

Reply:
Ok!